sites/main-site/src/content/blog/2025/survey-results-2025.mdx

```diff
@@ -0,0 +1,296 @@
+import fineuserrequest from "@assets/images/blog/survey-results-2025/nfcoresurvey2025-fineuserrequest.png";
+import finedevpositive from "@assets/images/blog/survey-results-2025/nfcoresurvey2025-finedevpositive.png";
+import finedevimprove from "@assets/images/blog/survey-results-2025/nfcoresurvey2025-finedevimprove.png";
+import finedevrequest from "@assets/images/blog/survey-results-2025/nfcoresurvey2025-finedevrequest.png";
+
+import { Image } from "astro:assets";
+
+## Introduction
+
+nf-core has a huge community of 11,000 members (based on Slack users), and this number does not seem appear to be diminishing.
+As the community gets bigger, it is getting harder to keep track of all the discussions happening across all of our Slack channels, and increasingly in face-to-face talks on Slack huddles and gather.town.
+Therefore the core team has decided to hold a yearly community survey to help provide us with an overview of the mood of community and in which ways the 'wind is blowing' in terms of needs and requirements.
+
+With this information, we plan to develop a roadmap for the continued development and progress the nf-core ecosystem and community over the next few years.
+
+## Key points
+
+- We had 209 responders, representing 1.8% of our Slack community.
+- nf-core has a responder NPS score of 54, representing a very positive opinion of nf-core, and are likely to recommend the community to colleagues and peers.
+- The community itself was the biggest positive aspect of nf-core, with responders finding it a welcoming, friendly, and helpful group of people.
+- While the existing amount of documentation was appreciated, it remains the biggest target for improvement primarily in terms of discoverability and consistency.
+
+## Methodology
+
+In mid-february 2025, we sent out on Slack a simple survey for all community members, regardless of whether they are users, developers, newcomers, or veterans of nf-core.
+This short 7 question survey asked the following main questions
+
+1. Community member type (user, developer, or both)
+2. Experience within nf-core (1 newcomer - 5 advanced)
+3. What country based in (country list)
+4. How likely to recommend nf-core (0-10 scale)
+5. What is liked the most about nf-core (free text)
+6. What difficulties have been encountered (free text)
+7. Any other feedback (free) text
+
+The survey was open for approximately a month, with one reminder halfway through.
+
+The answers from first four questions responses were aggregated and summarised as distributions.
+
+The free text answers from the latter three questions were assigned with one or more of various types of tags to group the response points into similar feedback topics.
+This tagging was independently performed by two core members and compared.
+
+Tagging was performed within Google Drive, and then all data was cleaned up, processed, and visualised R with the tidyverse set of packages.
```

Comment:
"the tidyverse set of packages" sounds a bit odd.

Reply:
It's what it is, change dit to collections for clarity